src/com/lftechnology/java/training/niraj/ems/controllers/EmployeeController.java

```diff
@@ -0,0 +1,127 @@
+package com.lftechnology.java.training.niraj.ems.controllers;
+
+import java.io.Console;
+import java.sql.SQLException;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.lftechnology.java.training.niraj.ems.domains.Employee;
+import com.lftechnology.java.training.niraj.ems.exceptions.CustomException;
+import com.lftechnology.java.training.niraj.ems.services.implementations.RouteServices;
+import com.lftechnology.java.training.niraj.ems.services.implementations.UserServiceImpl;
+import com.lftechnology.java.training.niraj.ems.utils.Constants;
+
+/**
+ * Controller related to logged in employee
+ * 
+ * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+ *
+ */
+public class EmployeeController {
+    private static final Logger LOGGER = Logger.getLogger(EmployeeController.class.getName());
+    private UserServiceImpl userService;
+
+    public EmployeeController() {
+        userService = new UserServiceImpl();
+    }
+
+    /**
+     * Adds an employee
+     * 
+     * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+     * @param loggedInUser
+     *            {@link Employee}
+     */
+    public void add(Employee loggedInUser) {
+        try (Scanner scanner = new Scanner(System.in)) {
+            Console console = System.console();
+            Employee employee = userService.addUser(scanner, console);
+            if (employee.getId() != null) {
+                LOGGER.log(Level.INFO, "Successfully added the user {0}", employee.getFullname());
+            }
+            RouteServices.routeAfterLogin(loggedInUser);
+        } catch (InputMismatchException | SQLException se) {
+            LOGGER.severe("There was some error. Please try again later.");
+            LOGGER.log(Level.SEVERE, "Exception:{0}", se);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Exception:{0}" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)